### PR TITLE
DE38831 - Update to latest my-courses

### PR DESCRIPTION
**Pull request by svanherk**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -16080,6 +16072,554 @@
...
+          "version": "1.2.9",
+          "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.9.tgz",
+          "integrity": "sha512-oeyj2H3EjjonWcFjD5NvZNE9Rqe4UW+nQBU2HNeKw0koVLEFIhtyETyAakeAM3de7Z/SW5kcA+fZUait9EApnw==",
+          "dev": true,
+          "optional": true,
+          "requires": {
+            "nan": "^2.12.1",
+            "node-pre-gyp": "^0.12.0"
+          },
+          "dependencies": {
```
Comment (AllanKerr) [inline package-lock.json:16085]: Shouldn't these all be getting flattened or am I mistaken?

Comment (svanherk) [reply package-lock.json:16085]: Yeah this is weird, I'll try rerunning

Comment (svanherk) [reply package-lock.json:16085]: That made things much worse lol 🤦

Comment (svanherk) [reply package-lock.json:16085]: Ok, at least back to what I had before.  Hmm 🤔

**Pull request merged**